msal/src/main/java/com/microsoft/identity/nativeauth/NativeAuthPublicClientApplication.kt

```diff
@@ -278,11 +283,6 @@ class NativeAuthPublicClientApplication(
         scopes: List<String>?,
         callback: SignInCallback
     ) {
-        LogSession.logMethodCall(
```

Comment:
The reason for having it here as well as in `suspend fun signIn()` is so we know whether the callback or coroutine method was called. You don't think that's useful information?

Reply:
Agreed. Callback or Coroutine is also something we want to gather for telemetry. The reason I removed them is because for callback method, there would be two repeating same logs.